.github/workflows/jacoco_check.yml

```diff
@@ -118,4 +118,4 @@ jobs:
               }
             }
 
-            core.setFailed('JaCoCo test coverage report generation failed, and related PR comments were updated.');
+            core.setFailed('JaCoCo test coverage report generation failed, and related PR comments were updated.');
```

Comment:
```suggestion
            core.setFailed('JaCoCo test coverage report generation failed, and related PR comments were updated.');

```

Reply:
Last empty is present in my version of file.